frontend/shared/utils/sidebarStartup.js

```diff
@@ -17,6 +17,11 @@ const toggleSidebarCollapse = function(isCollapsed, submit) {
         if (document.getElementById("sidebar-container") === null) {
             return;
         }
+        if ($("#sidebar-container").data("collapsed") == 'empty') {
+            if ($(document).width() < 500) {
+                toggleSidebarCollapse(true, false)
+            }
+        }
```

Comment:
this does work if the value would actually evaluate to "empty" (see below), but honestly it still takes a few seconds to kick in and hide the sidebar. I am not sure if this is a great solution even if the yesno filter worked correctly